sdk/src/main/java/com/meniga/sdk/models/transactions/TransactionsFilter.java

```diff
@@ -231,7 +241,11 @@ protected TransactionsFilter clone() throws CloneNotSupportedException {
 		Field skipField = null;
 		Field takeField = null;
 		for (Field member : mappableFields) {
-			if (member.getName().equals("isFiltering")) {
+			if (member.getName().equals("isFiltering") || member.getName().equals("includeMerchants")) {
```

Comment:
Shouldn't `member.getName().equals("includeMerchants")` go to the check below?

Reply:
No, we'd get "include=xxx" twice then.